.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
```

Comment:
Physical address makes a lot of sense ... but electronic address? For example, if I want to ping someone in a pull request, should I ask for permission before pinging them? (because pinging requires to expose the user electronic address on GitHub). And about the email, if that user publicly shows their email address in GitHub pull requests, should I also ask for permission to show that (already public) electronic address?

Reply:
It affects only private information, but i guess it should be more explicit by saying `non-public`?

Reply:
in general imho we should bring such feedback upstream. I would really prefer not forking the contents of this standard CoC.

Reply:
Not sure about that. Each project has different needs and ideas how to govern their community. Changing/tweaking/improving things might be OK. It's like bringing diversity to "diversity CoC".

Reply:
heh .. well I think CoC's are like software licenses .. yes each project is different but coming up with your own is likely going to create more problems than it solves.

Reply:
opened a ticket https://github.com/ContributorCovenant/contributor_covenant/issues/492